src/main/java/teammates/ui/pagedata/InstructorFeedbackResponseCommentsLoadPageData.java

```diff
@@ -95,8 +95,9 @@ private void init(FeedbackSessionResultsBundle bundle) {
             List<FeedbackResponseCommentAttributes> feedbackResponseCommentsAttributes,
             FeedbackQuestionAttributes question, FeedbackResponseAttributes response,
             String giverName, String recipientName, Map<FeedbackParticipantType, Boolean> responseVisibilities,
-            FeedbackSessionAttributes feedbackSession) {
+            FeedbackSessionAttributes feedbackSession, FeedbackSessionResultsBundle bundle) {
```

Comment:
Instead of passing the entire `bundle`, you can just pass the `instructorEmailNameTable`?